msg/servo_report.msg

```diff
@@ -0,0 +1,8 @@
+uint64 timestamp					# time since system start (microseconds)
+
+float32 servo_position					# feedback position of servo
+float32 servo_speed					# feedback speed of servo
+float32 servo_force					# force experienced by servo
+uint8 servo_id						# id of servo
+uint8 servo_function					# servo output function (one of Servo1...ServoN)
+
```

Comment:
Does Hitec populate power rating?

![Screenshot from 2022-06-21 11-10-31](https://user-images.githubusercontent.com/84712/174834564-f6a972ce-2467-45a6-9bb8-4b07cf38ffa4.png)

Reply:
Yes, Hitec provides a power rating, or a load percentage which the servo is experienced. I have updated as such